src/jdk.compiler/share/classes/com/sun/source/util/TreePath.java

```diff
@@ -61,34 +61,44 @@ public static TreePath getPath(TreePath path, Tree target) {
         Objects.requireNonNull(path);
         Objects.requireNonNull(target);
 
-        class Result extends Error {
-            static final long serialVersionUID = -5942088234594905625L;
-            @SuppressWarnings("serial") // Type of field is not Serializable
-            TreePath path;
-            Result(TreePath path) {
-                this.path = path;
+        class PathFinder extends TreePathScanner<TreePath,Tree> {
+            private TreePath result;
+
+
+            @Override
+            public TreePath scan(TreePath path, Tree docTree) {
```

Comment:
The second parameter has nothing to do with `DocTree`. Copy-paste?

Reply:
copy-paste error; will fix